### DISCUSSION TITLE
How do you pronounce ripgrep?

### DISCUSSION THREAD

**Discussion by Mehuge** [BurntSushi/ripgrep#2413] [General] [answered]
Is it rip-grep or r-i-p-grep?

**Comment by BurntSushi** [comment, marked as answer]
Related: https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#what-does-the-rip-in-ripgrep-mean

The "rip" in "ripgrep" is meant to me "to rip through the text very quickly." As in, "grip it and rip it" when playing golf. So that means pronouncing it is meant to be "rip-grep."

The "RIP" or "Rest In Peace" meaning just didn't dawn on me until someone pointed it out after the release. I was of course not unaware of the initialism, but I think the combination of my stylizing it in lowercase and only looking at it as a synonym for "fast" caused me to miss it.
